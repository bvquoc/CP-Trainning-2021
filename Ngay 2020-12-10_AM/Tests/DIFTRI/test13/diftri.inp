100
0 49
100 36
200 25
300 16
400 9
500 4
600 1
700 0
800 1
900 4
1000 9
1100 16
1200 25
1300 36
1400 49
1500 64
1600 81
1700 100
1800 121
1900 144
2000 169
2100 196
2200 225
2300 256
2400 289
2500 324
2600 361
2700 400
2800 441
2900 484
3000 529
3100 576
3200 625
3300 676
3400 729
3500 784
3600 841
3700 900
3800 961
3900 1024
4000 1089
4100 1156
4200 1225
4300 1296
4400 1369
4500 1444
4600 1521
4700 1600
4800 1681
4900 1764
5000 1849
5100 1936
5200 2025
5300 2116
5400 2209
5500 2304
5600 2401
5700 2500
5800 2601
5900 2704
6000 2809
6100 2916
6200 3025
6300 3136
6400 3249
6500 3364
6600 3481
6700 3600
6800 3721
6900 3844
7000 3969
7100 4096
7200 4225
7300 4356
7400 4489
7500 4624
7600 4761
7700 4900
7800 5041
7900 5184
8000 5329
8100 5476
8200 5625
8300 5776
8400 5929
8500 6084
8600 6241
8700 6400
8800 6561
8900 6724
9000 6889
9100 7056
9200 7225
9300 7396
9400 7569
9500 7744
9600 7921
9700 8100
9800 8281
9900 8464
