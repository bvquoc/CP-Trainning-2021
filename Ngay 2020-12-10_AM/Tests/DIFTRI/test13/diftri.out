12127400.0
